lib/charms/vault_k8s/v0/juju_facade.py

```diff
@@ -26,7 +26,7 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 3
```

Comment:
I think this whole file can be reverted now -- it's not used anymore.

Reply:
https://github.com/canonical/vault-k8s-operator/pull/534/commits/da289de9084fd2ffe309878323a2346afd3c537d